projects/ngx-img-cropper/package.json

```diff
@@ -4,5 +4,8 @@
   "peerDependencies": {
     "@angular/common": ">=8.0.0 <10.0.0",
     "@angular/core": ">=8.0.0 <10.0.0"
+  },
+  "dependencies": {
```

Comment:
I had `ng update` do this to my projects, too. It'd probably be better to move this to `peerDependencies`